core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -179,6 +179,19 @@ package object config {
   private[spark] val EVENT_LOG_CALLSITE_LONG_FORM =
     ConfigBuilder("spark.eventLog.longForm.enabled").booleanConf.createWithDefault(false)
 
+  private[spark] val EVENT_LOG_ENABLE_ROLLING =
+    ConfigBuilder("spark.eventLog.logRolling.enabled")
```

Comment:
Maybe, `spark.eventLog.logRolling.enabled` -> `spark.eventLog.rolling.enabled` since this is under `eventLog`?

Reply:
That's a good suggestion as it seems to sound redundant for using log twice. As @vanzin left review comment for this, would be nice to hear his thought as well before making change.

Reply:
I'm fine with Dongjoon's suggestion.